src/libertem/web/cli.py

```diff
@@ -5,17 +5,19 @@
 @click.command()
 @click.option('--port', help='port on which the server should listen on',
               default=9000, type=int)
+@click.option('--local_directory', help='local directory to manage dask-worker-space files',
```

Comment:
Nitpick: I would prefer a dash here instead of the underscore, following the usual unix conventions.

Reply:
Updated code to replace the dash.